accord-core/src/main/java/accord/messages/ReadData.java

```diff
@@ -256,7 +257,7 @@ public Iterable<TxnId> txnIds()
     @Override
     public Seekables<?, ?> keys()
     {
-        return readScope;
+        return Keys.EMPTY;
```

Comment:
We don't need the per-key data, it was for a C* impl reason that I determined was unnecessary - namely selecting a `nowInSeconds` that was consistent with the transaction's write `timestamp` (which is fine to be slightly stale, so can avoid bringing that info into memory)

Reply:
Note, I plan to add some stuff to this PR (or perhaps if you prefer a followup PR) to permit us to properly implement range transactions in the C* side. I had left it initially because range transactions are inherently more complicated for this element as they do not know which keys they want - they want ranges of keys, and this may be more info than we want/can bring into memory. So I have (almost) refactored this so that we do not need to declare the `keys` scope at all for `PreLoadContext` - we instead invoke an asynchronous method at the end of processing a message that provides the keys we need to operate over and it's up to the impl how it goes about that - which permits us to consistently operate over ranges as well.

Reply:
Ok, I'll revert those additional commits and file a new PR